docs/usage/configuration-options.md

```diff
@@ -2584,6 +2584,8 @@ Read the parser documentation at [breejs.github.io/later/parsers.html#text](http
 To parse Cron syntax, Renovate uses [@cheap-glitch/mi-cron](https://github.com/cheap-glitch/mi-cron).
 Renovate does not support scheduled minutes or "at an exact time" granularity.
 
+!!! note Actions triggered via the [Dependency Dashboard](https://docs.renovatebot.com/configuration-options/#dependencydashboard) are not restricted by a configured schedule.
```

Comment:
```suggestion
<!-- prettier-ignore -->
   !!! note
   Actions triggered via the [Dependency Dashboard](https://docs.renovatebot.com/configuration-options/#dependencydashboard) are not restricted by a configured schedule.
```

This should fix the Material for MkDocs admonition. 😉

Reply:
Well... this happens, if you copy & paste a sample from the GitHub's preview page instead of the raw display. 🙈